js/src/components/enhanced-conversion-tracking-settings/cta/pending-status.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import useAutoCheckEnhancedConversionTOS from '.~/hooks/useAutoCheckEnhancedConversionTOS';
+
+const PendingStatus = () => {
+	useAutoCheckEnhancedConversionTOS();
+
+	return null;
+};
```

Comment:
It's recommended to make the `useAutoCheckEnhancedConversionTOS` hook provide methods, and control whether to start polling or not in the `CTA` instead of controlling the hook's lifetime by whether it renders or not.

Reply:
Thanks for the recommendation @eason9487 . Can you review the updates made please?